plugins/plugin-orion/che-plugin-orion-editor/pom.xml

```diff
@@ -1,44 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This seems unrelated....what's happening here?

Reply:
Yeap, probably this is here occasionally